packages/@vue/cli-shared-utils/lib/babelHelpers.js

```diff
@@ -0,0 +1,39 @@
+const babel = require('@babel/core')
+
+const lib = exports.babelHelpers = {}
+
+lib.loadUsersConfig = (rootDir) => {
+  if (typeof rootDir !== 'string') {
+    throw new TypeError('`rootDir` must be a string')
+  }
+
+  const options = {
+    cwd: rootDir,
+    filename: 'package.json'
+  }
+
+  const partial = babel.loadPartialConfig(options)
```

Comment:
why use `loadPartialConfig` then manually normalize instead of just using [`loadOptions`](https://github.com/babel/babel/tree/master/packages/babel-core#babelloadoptionsoptions-object)?

Reply:
`loadPartialConfig` loads only users config. both `loadPartialConfig` and `loadOptions` return not valid config for passing it to the babel-loader(babel.transform), this why there is need manually normilize config.